src/apify/scrapy/requests.py

```diff
@@ -16,6 +16,7 @@
     ) from exc
 
 from crawlee import Request as CrawleeRequest
+from crawlee._types import HttpHeaders
```

Comment:
Just to make sure, this doesn't raise our minimum required crawlee version, right?

Reply:
Actually, it does, because in Scrapy integration we use conversion `HttpHeaders(...)` for headers. Thanks.